copado-function/app/Commit.config

```diff
@@ -5,7 +5,7 @@
 * [Description] (Commit Marketing Cloud changes) 
```

Comment:
Please add `installMcdevLocally` to this file

Reply:
Done